init.rb

```diff
@@ -91,6 +91,11 @@ class << Setting
 FullTextSearch::TagType
 FullTextSearch::Type
 
+if Gem::Version.new(Redmine::VERSION) > Gem::Version.new("5.1")
```

Comment:
I added this method because the original `Issue#sql_for_any_searchable_field`is called and cause the error by the time we implement the `IssueAnySearchable#sql_for_any_searchable_field`.

Reply:
Can we use feature based check instead of version based check?

BTW, should we use `>=` here?

Reply:
fix: [9cfbe32](https://github.com/clear-code/redmine_full_text_search/pull/164/commits/9cfbe32778977409cdd55091fc60deb8394d24b6) Thanks, is this implementation satisfied with your feature based check?